Rakefile

```diff
@@ -25,14 +25,18 @@ require 'rspec/core'
 require 'rspec/core/rake_task'
 
 desc "Run all specs in spec directory (excluding plugin specs)"
-RSpec::Core::RakeTask.new(:spec => 'app:db:test:prepare')
```

Comment:
This caused concurrency issues and I couldn't notice any difference after removing it.